### ISSUE TITLE
upgrade error

### ISSUE THREAD

**Issue by DrMcStrange** [bcbio/bcbio-nextgen#2582]
I'm getting the following error when I try to upgrade:

> ERROR conda.core.link:_execute(502): An error occurred while uninstalling package 'bioconda::bioconductor-genomeinfodbdata-1.0.0-r3.4.1_1'.
> UnicodeDecodeError('ascii', '+ R CMD REMOVE --library=/gd/apps/bcbio/anaconda/lib/R/library/ GenomeInfoDbData\nError in find.package(pkgs, lib) : \n  there is no package called \xe2\x80\x98GenomeInfoDbData\xe2\x80\x99\nExecution halted\n', 146, 147, 'ordinal not in range(128)')
> Attempting to roll back.

I'm just using the standard command `bcbio_nextgen.py upgrade -u stable --tools --data`

Any ideas?

**Comment by chapmanb** [comment]
Sorry about the problems. It looks like something went funny with the conda install of that package and there are some extra problematic characters in the unlink file that cause it fail. I'm not sure how that happened, but you could manually edit the problem file:
```
/gd/apps/bcbio/anaconda/pkgs/bioconductor-genomeinfodbdata-1.0.0-r3.4.1_1/bin/.bioconductor-genomeinfodbdata-pre-unlink.sh
```
to remove the unicode characters around `GenomeInfoDbData` on the single line in that file. Then it should hopefully uninstall cleanly for you. Hope this helps.

**Comment by chapmanb** [comment]
Bennet -- thanks for trying to work through this. I'm not sure what is going on here but you try removing that file or the line from the file to skip the removal command altogether. The cleanup should not be a big deal and the new version could install on top of it. Hope this gets you through whatever is going on here.

**Comment by DrMcStrange** [comment]
Okay, I've tried removing the file to no avail, and likewise the whole directory `/gd/apps/bcbio/anaconda/pkgs/bioconductor-genomeinfodbdata-1.0.0-r3.4.1_1/`

At this point I think I'll just nuke the whole thing and re-install fresh.

**Comment by chapmanb** [comment]
Bennet;
Sorry this is still causing issues and the previous ideas didn't help. I was able to find a system where we could reproduce this issue and I believe the underlying problem is that the underlying library has just been removed or is not present for some reason. I'm still not sure what causes this, but the workaround is to comment out the `R CMD REMOVE` line in the unlink file that gets copied into the conda environment:
```
/path/to/bcbio/anaconda/bin/.bioconductor-bsgenome.hsapiens.ucsc.hg19-pre-unlink.sh
```
Hope this catches you before doing a full install, and thanks again for the patience debugging.
